ex01.exs

```diff
@@ -75,13 +73,26 @@ defmodule Ex01 do
 
   def counter(value \\ 0) do
     # ...your code
+	receive do
+		{:next, from} ->
+			send from, {:next_is, value}
+			counter(value + 1)
+		{:next_is, value} ->
+			counter(value)
+	end
   end
 
   def new_counter(initial_value \\ 0) do
     # ... your code
+	count = spawn Ex01, :counter, [initial_value]
```

Comment:
This reports _variable "count" is unused_ during compilation. I treat warnings as errors, and I recommend you do too. That's based on experience: when I ignore them, I typically find that I end up debugging back to that exact location.

# -1